rest.insomnia.client.yml

```diff
@@ -0,0 +1,64 @@
+app-id: rest.insomnia.client
+runtime: org.freedesktop.Sdk
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+rename-desktop-file: insomnia.desktop
+rename-icon: insomnia
+separate-locales: false
+command: insomnia
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  # I'm not a fan of this permission, but for some reason Insomnia won't start
+  # without it. Contributions welcome:
+  - --device=all
+modules:
+  - name: insomnia
+    buildsystem: simple
+    build-options:
+      strip: false
+      no-debuginfo: true
+      no-debuginfo-compression: true
+    build-commands:
+      - tar xf insomnia.tar.gz
+      - install -dm755 /app/share/
+      - install -dm755 /app/main/
+      - mv Insomnia.Core-2021.3.0/* /app/main
+
+      - install -Dm644 insomnia.desktop /app/share/applications/insomnia.desktop
+      - install -Dm644 insomnia.png /app/share/icons/hicolor/scalable/apps/insomnia.png
+
+      - install -Dm755 insomnia.sh /app/bin/insomnia
+      - install -Dm644 rest.insomnia.client.appdata.xml /app/share/metainfo/$FLATPAK_ID.appdata.xml
+    sources:
+      - type: file
+        dest-filename: insomnia.tar.gz
+        only-arches:
+          - x86_64
```

Comment:
Please add https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64 to limit architectures.